File: CalculoVenta.py
# Función para calcular el total a pagar
def calcular_total():
    # Pedir al usuario el precio del artículo
    precio = float(input("Ingrese el precio del artículo: "))

    # Pedir al usuario la cantidad de artículos que lleva
    cantidad = int(input("Ingrese la cantidad de artículos que lleva: "))

    # Calcular el total a pagar
    total = precio * cantidad

    # Mostrar el total que debe abonar el cliente
    print(f"El total a pagar por {cantidad} artículo(s) es: ${total:.2f}")

# Llamar a la función
calcular_total()    

File: prog_java/CalculoVenta.java
package prog_java;
import java.util.Scanner;

public class CalculoVenta {

    public static void main(String[] args) {
        // Crear un objeto Scanner para leer la entrada del usuario
        Scanner scanner = new Scanner(System.in);
        
        // Pedir el precio del artículo al usuario
        System.out.print("Ingrese el precio del artículo: ");
        double precio = scanner.nextDouble();
        
        // Pedir la cantidad de artículos al usuario
        System.out.print("Ingrese la cantidad de artículos que lleva el cliente: ");
        int cantidad = scanner.nextInt();
        
        // Calcular el total a pagar
        double total = precio * cantidad;
        
        // Mostrar el total al comprador
        System.out.println("El total a abonar por el comprador es: $" + total);
        
        // Cerrar el scanner
        scanner.close();
    }
}



